tests/bootstrap.php

```diff
@@ -1,7 +0,0 @@
-<?php
-
-$loader = @include __DIR__ . '/../vendor/autoload.php';
-if (!$loader) {
-    $loader = require __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
This LGTM (IMO) :+1: 

Note however that removing this condition means we can no longer run the tests if this is installed as a composer dependency.

Personally, I would consider this a non-issue, but this has been raised before: https://github.com/reactphp/dns/pull/24#issuecomment-117041513